lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3667,6 +3667,28 @@ void SwiftASTContext::CacheModule(swift::ModuleDecl *module) {
   m_swift_module_cache.insert({ID, module});
 }
 
+void SwiftASTContext::MapModuleABINameToRealName(swift::ModuleDecl *module) {
```

Comment:
The function is called Map, but it actually just enters something into the map. How about `RegisterModuleABIName()`?